docs/tests/sals/nettools/test_nettools.md

```diff
@@ -0,0 +1,610 @@
+### test_01_tcp_connection_test_to_public_ipv4_succeed[1.1.1.1-53-5]
```

Comment:
I can see that the scenarios are repeated because of this issue #551, it is fixed on the development branch, can you merge it into your branch and regenerate them again?

Reply:
done